x-pack/plugin/downsample/src/main/java/org/elasticsearch/xpack/downsample/DownsampleMetrics.java

```diff
@@ -30,6 +30,7 @@
 public class DownsampleMetrics extends AbstractLifecycleComponent {
 
     public static final String LATENCY_SHARD = "es.tsdb.downsample.latency.shard.histogram";
+    public static final String LATENCY_MASTER = "es.tsdb.downsample.latency.master.histogram";
```

Comment:
The `master` part is leaking implementation details that may change in the future. I think that maybe api latency or operation latency is a better name here. Wdyt?

Reply:
Switched to `total`.